Gruntfile.js

```diff
@@ -33,7 +33,7 @@ module.exports = function (grunt) {
             angular: {
                 src: ['bower_components/angular/angular.min.js',
                       'bower_components/angular-resource/angular-resource.min.js',
-                      'bower_components/angular-route/angular-route.min.js',
+                      'node_modules/angular-ui-router/release/angular-ui-router.min.js',
```

Comment:
What about installing all dependencies by npm?
Webpack will be happier once we use it: https://webpack.github.io/docs/usage-with-bower.html#prefer-modules-from-npm-over-bower